### PR TITLE
Problems

### PR DESCRIPTION
**Pull request by MeryVar**

### CODE REVIEW

File: 328-340.ipynb [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+    }
+   ],
+   "source": [
+    "def prob_329(input_list, index):\n",
+    "    if len(input_list) >= index:\n",
+    "        sum_num = 0\n",
+    "        for i in range(index - 1):  \n",
+    "            sum_num += input_list[i]\n",
+    "        return sum_num\n",
+    "    return sum([num for num in input_list if len(input_list) >= index])\n",
```
Comment (levon2111) [inline 328-340.ipynb:51]: Can you please explain why when len(input_list) <  index we are returning return sum([num for num in input_list if len(input_list) >= index]) ?

**Review by levon2111 — CHANGES_REQUESTED** [review]

File: 328-340.ipynb (hunk 1 above)
Comment (MeryVar) [reply 328-340.ipynb:51]: Thank you for the review!
At first I tried to use list comprehension, then I decided to separate. The last return should be deleted, I forgot to delete it.

File: 341-350.ipynb [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+     "execution_count": 120,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "def prob_349(input_list):\n",
+    "    odd_indices = []\n",
+    "    for index, num in enumerate(input_list):\n",
+    "        if isinstance(num, int) and num % 2 == 1:\n",
```
Comment (levon2111) [inline 341-350.ipynb]: Can you please check the problem once again ? You need to convert items to integer.

File: 341-350.ipynb [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+    "            odd_indices.append(index)\n",
+    "    if odd_indices:\n",
```
Comment (levon2111) [inline 341-350.ipynb:237]: What should happen when in list we don't have odd numbers ?

File: 341-350.ipynb [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+   ],
+   "source": [
+    "def prob_350(input_list):\n",
+    "    positive_count = 0\n",
+    "    sum_sqr = 0\n",
+    "    for num in reversed(input_list):\n",
+    "        if num > 0:\n",
+    "            positive_count += 1\n",
+    "            sum_sqr += num ** 2\n",
+    "        if positive_count == 5:\n",
```
Comment (levon2111) [inline 341-350.ipynb]: Why we have this condition here ? In any case we are breaking the iteration .

**Pull request merged**